### PR TITLE
JBPM-4833 - Data Modeller MVP, Tests, and related improvments

### PR DESCRIPTION
**Pull request by wmedvede**
-DataObjectBrowser, MPV improvements + tests + i18n review

### CODE REVIEW

**Comment by wmedvede** [comment]
@pefernan 

Hello Pere, would you mind review this PR?

Thanks

**Comment by pefernan** [comment]
+1

File: kie-wb-common-screens/kie-wb-common-data-modeller/kie-wb-common-data-modeller-client/src/main/java/org/kie/workbench/common/screens/datamodeller/client/util/ObjectPropertyComparator.java [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+ * <p/>
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
```
Comment (psiroky) [inline kie-wb-common-screens/kie-wb-common-data-modeller/kie-wb-common-data-modeller-client/src/main/java/org/kie/workbench/common/screens/datamodeller/client/util/ObjectPropertyComparator.java]: License header should not contain HTML tags.

**Comment by wmedvede** [comment]
No idea how this could be added, btw, fixed.

**Comment by csadilek** [comment]
6.4.x: 78a8e665131c3fc31a360ac3085561bba1832560

@wmedvede please don't forget to send this to master ;).

**Pull request closed without merging**